gnomad/resources/grch38/reference_data.py

```diff
@@ -42,6 +42,12 @@ def _import_clinvar(**kwargs) -> hl.Table:
     return clinvar
 
 
+def _import_clinvar_pathogenic(**kwargs) -> hl.Table:
```

Comment:
If we did the above suggestion we actually wont need this anymore, though `_import_clinvar` would now call the function for annotating clinvar with pathogenic information